### PR TITLE
Add support for LDO Picobilical MCU

### PR DESCRIPTION
**Pull request by ndanyluk**
The LDO Picobilical mod is an umbilical mod that's becoming more and more common as a standalone V0 mod.  Based on the [V0-Umbilical mod](https://github.com/VoronDesign/Voron-Hardware/tree/master/V0-Umbilical), this LDO toolhead & frame board combo adds an RP2040 MCU to the V0 beyond the main microcontroller.  This mod is included with the V0-S1 line of LDO V0 kits (both V0.2-S1 & V0.1-S1).

Original docs can be found at [LDO's Documentation Site](https://docs.ldomotors.com/en/voron/voron01/Picobilical).  Additionally, [schematics](https://github.com/MotorDynamicsLab/LDO-Picobilical/blob/master/Hardware/Frame1.3/Picobilical_A1.3_Schematics.pdf) and [Klipper config](https://github.com/MotorDynamicsLab/LDO-Picobilical/blob/master/Klipper_Configs/ldo-picobilical.cfg) are provided on Github

This PR is still a WIP and needs some guidance on how to add the unique features of this board (2 Neopixel ports that are independent of eachother, a 2 pin filament runout sensor(?) and a small buzzer/speaker/beeper).

### CODE REVIEW

**Comment by Surion79** [comment]
don't assign board defaults, if they are not usable in klippain. this is manual work for users of for e.g. a beeper function.
Because when klippain adds functionality, it will define a a generic pin which then has to be assigned in the defaults.

Please keep the definition and the defaults, which are currently usable in klippain in this PR and set for each missing feature a feature request in the issues tab. 
Then we can track and see what to implement next.

**Comment by ndanyluk** [comment]
@Surion79 gotcha that makes sense!

I'll remove the pieces that are currently not supported in klippain (i.e. 2nd neopixel, beeper, etc) and fix the PR.  Thanks!!

**Comment by Surion79** [comment]
@ndanyluk Do you think we can merge it? it is still in draft

**Comment by ndanyluk** [comment]
Hi @Surion79, sorry for the enormous delay.  This has sat for way too long.

Updated the Picobilical config to follow Klippain feature set (i.e., only one neopixel, MCU names for all the pins, etc).

I'd like to follow this up with a second PR that adds my macro overrides for the neopixels so that users can benefit from the two neopixel ports (greatly simplifies wiring), but this config has been working on my V0.2 for months without issue!

File: user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#### LDO Picobilical Frame PCB A1.3 definition ####
+#-------------------------------------------------#
+
+[mcu toolhead]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+restart_method: command
+##--------------------------------------------------------------------
+
+[include config/mcu_definitions/toolhead/LDO_Picobilical_Frame_vA1.3.cfg] # Do not remove this line
```
Comment (logu) [inline user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg:11]: Hello 
I noticed that the the path to the config file was different : 
[include config/mcu_definitions/toolhead/LDO_Picobilical_vA1.3.cfg] 
regards

Comment (logu) [reply user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg:11]: Or maybe this file: LDO_Picobilical_Frame_vA1.3.cfg hasn't been committed. I am trying to get this to work on my Voron v0 built using the LDO kit and I am getting the following error : 
`
Option 'aliases' in section 'board_pins picobilical_mcu' must have 2 elements

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted
`

File: user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+[board_pins picobilical_mcu]
+mcu: toolhead
+aliases:
+    X_STOP=MCU_XES
```
Comment (logu) [inline user_templates/mcu_defaults/toolhead/LDO_Picobilical_vA1.3.cfg:15]: Hello again here a coma is missing : ` X_STOP=MCU_XES ,`

**Pull request closed without merging**